lib/membrane_webrtc/signaling_channel.ex

```diff
@@ -1,219 +1,42 @@
 defmodule Membrane.WebRTC.SignalingChannel do
   @moduledoc """
-  Channel for sending WebRTC signaling messages between Membrane elements
-  and other WebRTC peers.
-
-  The flow of using the signaling channel is the following:
-  - Create it with `new/0`.
-  - Register the peer process (the one to send and receive signaling messages)
-    with `register_peer/2`.
-  - Pass the signaling channel to `Membrane.WebRTC.Source` or `Membrane.WebRTC.Sink` (this
-    can also be done before the call to `register_peer/2`).
-  - Send and receive signaling messages. Messages can be sent by calling `signal/2`.
-    The channel sends `t:message/0` to the peer.
+  Deprecated since v0.24.0. Use #{inspect(Membrane.WebRTC.Signaling)} instead.
   """
   use GenServer
 
   require Logger
 
-  alias ExWebRTC.{ICECandidate, SessionDescription}
-
-  @enforce_keys [:pid]
-  defstruct @enforce_keys
-
-  @type t :: %__MODULE__{pid: pid()}
-
-  @typedoc """
-  Messages sent by the signaling channel to the peer.
-  """
-  @type message :: {__MODULE__, pid(), message_content, metadata :: map}
```

Comment:
Now I'm a bit concerned about these types, `message` in particular. Previously they were `{SignalingChannel, `, now they are `{Signaling, }`, so it may break people's code :/

Reply:
Do you think about Boombox, or about the community?

Reply:
Should we really care about it? The minor is bumped and I guess that the only things that use the messages in the form of `{Signaling | SignalingChannel, pid, ...}` are Boombox and WebRTC components that I am currently working on

Reply:
Right, I agree